ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,58 @@
+# Issue Templates
+
+## Bug
+
+Make sure to add **all the information needed to understand the bug** so that
+someone can help. If the info is missing we'll add the 'Needs more information'
+label and close the issue until there is enough information. Use the template
+below:
+
+---------------------
+
+### Subject
+
+Describe your issue here.
+
+### Environment
+
+* version of Python
+* what `pip freeze` says
+* OS where the client is running
+* ManageIQ version
+
+### Steps to Reproduce
+
+Tell us how to reproduce this issue. Please provide a working demo if possible.
+
+### Expected Behaviour
+
+Tell us what should happen
+
+### Actual Behaviour
+
+Tell us what happens instead
+
+---------------------
+
+## Feature Request
+
+In order to prioritize our feature requests and be able to clearly discuss the
+benefits of developing a feature, please follow the template below. If the info
+is missing we'll add the 'Needs more information' label and close the issue
+until there is enough information.
+
+---------------------
+
+### Summary
+
+Describe you would like to acheive
```

Comment:
Summary section it would be better to state:
Describe what you like to achieve (missing what and achieve spelled incorrectly).

Importance section, I think it would be better to state:
Tell us why you believe this feature request is important.

Use Cases:
I think it would be simpler to just remove the word 'few' or change to 'a few' instead of few (just my opinion)

Reply:
I also agree here.